sqlite3.go

```diff
@@ -23,6 +23,18 @@ import (
 	"github.com/go-rel/sql/builder"
 )
 
+type sqlAdapter struct {
+	*sql.SQL
+}
+
+// Name of database type this adapter implements.
+const Name string = "sqlite3"
```

Comment:
I think we should stick with Name or DBType only?

Reply:
Do you mean to rename this same as func name?

Reply:
yes, or the otherway around is fine too
since we want to use this constant for comparison too, so better to be consistent